css/pages/index.css

```diff
@@ -19,3 +55,20 @@
     font-size: 1rem;
   }
 }
+
+@media screen and (max-width: 430px) {
+  .container {
+    max-width: 350px;
+    font-size: 0.7em;
+  }
+  .card {
+    width: 5rem;
+    padding: 2px 0px;
+  }
+  .card a {
+    font-size: 8px;
```

Comment:
re: font-size

Reply:
> re: font-size

Yeah I will change px with rem in font-size. Should I do that for padding too ?

Reply:
If we use `rem` for padding and if some user changes default font-size from 16px to say 20px, our padding will increase which will result in breaking of our design.

Example: 
`padding: 2rem` => in default state when the font-size is 16px, padding will be 32px.
If a user changes their default font-size to 20px, the padding will increase to 40px which might break the design if we're having things closely lined up against each other.


We have to decide which properties should adapt to changes like screen width/font-size and which properties should stay fixed irrespective of variables.

Hope that clears your doubt! 😄

Reply:
> If we use `rem` for padding and if some user changes default font-size from 16px to say 20px, our padding will increase which will result in breaking of our design.
> 
> Example:
> `padding: 2rem` => in default state when the font-size is 16px, padding will be 32px.
> If a user changes their default font-size to 20px, the padding will increase to 40px which might break the design if we're having things closely lined up against each other.
> 
> We have to decide which properties should adapt to changes like screen width/font-size and which properties should stay fixed irrespective of variables.
> 
> Hope that clears your doubt! 😄

Yep thank u so much :)